eccehomo/test_settings.py

```diff
@@ -0,0 +1,4 @@
+import os
+
+TEST_MEDIA_ROOT = os.path.join(os.getcwd(), 'test_media')
```

Comment:
Instead of using the current path for test media, I would use a folder inside of `/tmp` which is disposable.